app/Http/Controllers/PostController.php

```diff
@@ -24,9 +24,7 @@ public function store(Request $request): RedirectResponse
         $post = new Post;
         $post->title = $request->input('title');
         $post->body = $request->input('body');
-
         $post->save();
```

Comment:
Al momento de crear la publicación, hace falta guardar el id del usuario que está conectado, para saber luego a quién pertenece el post.

Puedes hacerlo usando el helper `auth()`:
https://laravel.com/docs/9.x/helpers#method-auth
```php
$post->user_id = auth()->user()->id;
```

O usando el facade `Auth`:
https://laravel.com/docs/9.x/authentication#retrieving-the-authenticated-user
```php
use Illuminate\Support\Facades\Auth;
...

$post->user_id = Auth::id();
```

Reply:
ok